label_sync/main.go

```diff
@@ -206,17 +207,14 @@ func copyStringMap(originalMap map[string]string) map[string]string {
 // Ensures the config does not duplicate label names
 func (c Configuration) validate() error {
 	// Check default labels
-	seen := make(map[string]string)
-	if err := validate(c.Default.Labels, "default", seen); err != nil {
+	seen, err := validate(c.Default.Labels, "default", make(map[string]string))
+	if err != nil {
 		return fmt.Errorf("invalid config: %v", err)
 	}
 	// Check other repos labels
 	for repo, repoconfig := range c.Repos {
```

Comment:
this is where you can warn if you bump into a a repo whose prefix is not present in `strings.Split(*orgs, ",")`